test/parallel/test-http-client-abort-response-event.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+const server = http.createServer(function(req, res) {
+  res.end();
+});
+
+server.listen(0, common.mustCall(function() {
+  const req = http.request({
+    port: this.address().port
+  }, common.mustCall());
+
+  req.on('abort', common.mustCall(function() {
+    server.close();
+  }));
+
+  req.end();
+  req.abort();
+  req.abort();
```

Comment:
Havn't checked this completely, but two `req.abort`s are intentional?

Reply:
/ping @tadhgcreedon Two abort() calls like that are intentional? If so, great. If not, can you remove one? Or I can totally remove one. Either way. I just want to see this get merged....